guides/source/generators.md

```diff
@@ -52,7 +52,7 @@ class InitializerGenerator < Rails::Generators::Base
 end
 ```
 
-NOTE: `create_file` is a method provided by `Thor::Actions`. Documentation for `create_file` and other Thor methods can be found in [Thor's documentation](https://rdoc.info/github/erikhuda/thor/master/Thor/Actions.html)
+NOTE: `create_file` is a method provided by `Thor::Actions`. Documentation for `create_file` and other Thor methods can be found in [Thor's documentation](https://rdoc.info/github/erikhuda/thor/main/Thor/Actions.html)
```

Comment:
Thor is now hosted at https://github.com/rails/thor, instead of https://github.com/erikhuda/thor.  What about linking to the gem's version of documentation?

```suggestion
NOTE: `create_file` is a method provided by `Thor::Actions`. Documentation for `create_file` and other Thor methods can be found in [Thor's documentation](https://rdoc.info/gems/thor/Thor/Actions).
```